bftclient/include/bftclient/bft_client.h

```diff
@@ -44,7 +44,7 @@ class Client {
   // Throws a BftClientException on error.
   Reply send(const WriteConfig& config, Msg&& request);
   Reply send(const ReadConfig& config, Msg&& request);
-  std::deque<Reply> sendBatch(std::deque<WriteRequest>& write_requests, const std::string& cid);
+  std::unordered_map<uint64_t, Reply> sendBatch(std::deque<WriteRequest>& write_requests, const std::string& cid);
```

Comment:
unordered_map does not have a specific order of elements. Will the client pool know to match replies to requests?

Reply:
Yes, before we send the batch, we create a dictionary between the sequence number and the CID.